ynr/apps/uk_results/views/base_views.py

```diff
@@ -37,6 +37,12 @@ def get_form_kwargs(self):
             voting_system=Ballot.VOTING_SYSTEM_FPTP,
             ballot_paper_id=self.kwargs["ballot_paper_id"],
         )
+
+        if not self.ballot.polls_closed:
+            raise Http404(
+                f"Polls are still open for {self.ballot.ballot_paper_id}"
+            )
```

Comment:
I wonder if we should add "Please check back after polls close at {{ closing_time }}."